pkg/operator/kube_cloud_config/bootstrap.go

```diff
@@ -0,0 +1,86 @@
+package kube_cloud_config
```

Comment:
ugly package name with underscore.

Reply:
I have renamed the package to kubecloudconfig.